### ISSUE TITLE
Re-introduce Point addition to sites

### ISSUE THREAD

**Issue by fbl773** [fbl773/point-b_ank#38]
# ~Task - Complete the Point Creation workflow conversion
~"User Story": #28 
## Objective
Bring the Projectile point creation back to its previous functionality using the new API from #4 

### 3.2 Prerequisites 
- [x] Add missing db fields `Length, Width, Height`
- [x] Add missing db field  `period_id` for cases where only period can be determined. ~meet
- [x] Add missing db field `location`
- [x] Write a `PointCard.jsx` file similar to `SiteCard.jsx` for `ProjectileList.jsx` 

### 3.3 TODO
- [x] Generate name from `_id` and perhaps `culture`
- [ ] Create a new issue. Maybe "Patternize Pass 2" for `<T>List.jsx`'s and Whatever we could call `Site.jsx` 
and `Catalogue.jsx`.
- [x] Add missing form field `cross_section`
- [x] Acquire Available Periods (Dependent on #38)
- [x] Acquire Available Cultures  
- [x] Acquire Available Materials

**Comment by fbl773** [comment]
Related to #50

**Comment by fbl773** [comment]
If you don't get to it now, we should make it such that a projectile point can be moved to a different site (in case of errors). This is for a future item though...

**Comment by fbl773** [comment]
@Flake-bender I've got a few questions for you:
1. Would it be valuable to have both a `Period` and `Culture` field on points? My original thought was that if we cannot identify a culture, we should not guess at a period - but I'm also not an archaeologist!
2. Presently, we have dimension fields for length, width, and height. Is height relevant? Should we keep it?

**PSA** - the server went down at pblank.blewi.xyz and I'm not entirely sure why. I'm going to try to get the development version of the app up there shortly, but I'd like it to be more feature-complete. Projectile Points are essentially the last thing We need to handle before it has its original functionality. The code is looking much **much** cleaner. :) Also hope your summer is going well!

**Comment by fbl773** [comment]
@DanielleDPowell - Feel like tackling any of those check-items? I was thinking about assigning the DB pre-requisites to you.

**Comment by Flake-bender** [comment]
> @Flake-bender I've got a few questions for you:
> 
> 1. Would it be valuable to have both a `Period` and `Culture` field on points? My original thought was that if we cannot identify a culture, we should not guess at a period - but I'm also not an archaeologist!
> 2. Presently, we have dimension fields for length, width, and height. Is height relevant? Should we keep it?
> 
> **PSA** - the server went down at pblank.blewi.xyz and I'm not entirely sure why. I'm going to try to get the development version of the app up there shortly, but I'd like it to be more feature-complete. Projectile Points are essentially the last thing We need to handle before it has its original functionality. The code is looking much **much** cleaner. :) Also hope your summer is going well!

Yes, sometimes it is possible to ID the time period in which the point was made, but not the specific culture. This happens all the time actually, especially if the point is broken/fragmentary, with some diagnostic features missing.

The dimensions should be "length, width, thickness", and yes, thickness is important, especially for diagnosing certain early period lanceolate points.
